client/components/seo-preview-pane/index.jsx

```diff
@@ -117,25 +161,29 @@ export class SeoPreviewPane extends PureComponent {
 						</p>
 					</div>
 					<VerticalMenu onClick={ this.selectPreview }>
-						<SocialItem service="google" />
-						<SocialItem service="facebook" />
-						<SocialItem service="twitter" />
+						{ compact( [
+							post && 'wordpress',
+							'google',
+							'facebook',
+							'twitter'
+						] ).map( service => <SocialItem { ...{ key: service, service } } /> )
+						}
 					</VerticalMenu>
 				</div>
 				<div className="seo-preview-pane__preview-area">
 					<div className="seo-preview-pane__preview">
 						{ post && get( {
+							wordpress: PreviewReader( site, post ),
 							facebook: FacebookPost( site, post ),
-							google: GooglePreview( site ),
+							google: PreviewGoogle( site ),
```

Comment:
Oh, I think the naming of these have changed quite a few times in the other PRs. It used to be `PreviewFacebook`, `PreviewTwitter`, etc so I was renaming this to match it up with the others.